### PR TITLE
Fix #2607 modification of logind source

### LINKED ISSUE #2607: Building Mir modifies the logind source
```
$ git status
On branch main
Your branch is up to date with 'origin/main'.

nothing to commit, working tree clean

$ cmake -DCMAKE_BUILD_TYPE=Debug -Bbuild-Debug -H. -GNinja

$ cmake --build build-Debug

# ...

$ git status   
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   src/server/console/logind-seat.c
        modified:   src/server/console/logind-seat.h
        modified:   src/server/console/logind-session.c
        modified:   src/server/console/logind-session.h

no changes added to commit (use "git add" and/or "git commit -a")
```

### PR DESCRIPTION
**Pull request by graysonguarino**
Fixes #2607. I believe this should tackle it as I cannot reproduce the error now, but this should be double-checked by someone who faces this bug.

### CODE REVIEW

File: src/server/console/CMakeLists.txt [hunk 1]
```diff
@@ -9,7 +9,13 @@ add_library(
   logind-seat.c                             logind-seat.h
   logind-session.c                          logind-session.h
```
Comment (Saviq) [inline src/server/console/CMakeLists.txt:10]: These should go away, then? Are they even used? If we treat them the same as wayland protocols, we shouldn't have those checked in at all, but instead rely on the generated ones.

File: src/server/console/CMakeLists.txt [hunk 2]
```diff
@@ -6,10 +6,14 @@ add_library(
   linux_virtual_terminal.cpp                linux_virtual_terminal.h
   logind_console_services.cpp               logind_console_services.h
   minimal_console_services.h                minimal_console_services.cpp
-  logind-seat.c                             logind-seat.h
-  logind-session.c                          logind-session.h
```
Comment (AlanGriffiths) [inline src/server/console/CMakeLists.txt:10]: These files should also be removed from git

**Review by AlanGriffiths — CHANGES_REQUESTED** [review]
Mostly good. Just remove the files that should never have been in source control from source control

**Comment by AlanGriffiths** [comment]
```plain
$ git diff
diff --git a/tests/unit-tests/CMakeLists.txt b/tests/unit-tests/CMakeLists.txt
index 1a478a82b2..de841283dc 100644
--- a/tests/unit-tests/CMakeLists.txt
+++ b/tests/unit-tests/CMakeLists.txt
@@ -32,6 +32,7 @@ include_directories(
   ${PROJECT_SOURCE_DIR}/src/include/gl
   ${PROJECT_SOURCE_DIR}/src/platforms/common/client
   ${PROJECT_SOURCE_DIR}/src/platforms/common/server
+  ${CMAKE_BINARY_DIR}/src/server/console
   ${GLIB_INCLUDE_DIRS}
   ${GIO_INCLUDE_DIRS}
 )
```

File: src/server/console/CMakeLists.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 include_directories(
   ${DRM_INCLUDE_DIRS}
-  ${CMAKE_CURRENT_BINARY_DIR}
+  ${CMAKE_CURRENT_BINARY_DIR}/src/server/console
```
Comment (AlanGriffiths) [inline src/server/console/CMakeLists.txt:3]: This _was_ right before the last commit

**Comment by AlanGriffiths** [comment]
@graysonguarino

1. Revert or drop the last commit
2. Copy the above patch to the clipboard
3. In Clion: Git/Patch/Apply patch from Clipboard
4. Build locally
5. Commit & push

**Review by AlanGriffiths — APPROVED** [review]
LGTM

@Saviq WDYT?

**Review by Saviq — APPROVED** [review]
Yup!

bors r=AlanGriffiths,Saviq

**Comment by AlanGriffiths** [comment]
bors retry

**Pull request merged**